### PR TITLE
Add option to request GEFLITST member for activity

### PR DESCRIPTION
**Pull request by Staartvin**
Fixes issue #660

### CODE REVIEW

File: module/Activity/src/Activity/Service/Activity.php [hunk 1]
```diff
@@ -282,11 +283,19 @@ protected function generateActivity(array $params, $user, $organ, $dutch, $engli
...
         $em->flush();
 
-        //$this->getEmailService()->sendActivityCreationEmail($activity);
+        // Send an email when a new Activity was created.
         $this->getEmailService()->sendEmail('activity_creation', 'email/activity',
             'Nieuwe activiteit aangemaakt op de GEWIS website | New activity was created on the GEWIS website',
             ['activity' => $activity]);
 
+        // Send email to GEFLITST if user checked checkbox of GEFLITST
+        if ($activity->getRequireGEFLITST()) {
```
Comment (jszanto) [inline module/Activity/src/Activity/Service/Activity.php]: Doing this here will also send out an email everytime the activity is updated, which probably isn't that handy.

File: module/Activity/view/email/activity_created_require_GEFLITST.phtml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Er is een nieuwe activiteit aangemaakt op de GEWIS website waarvoor een fotograaf van GEFLITST nodig is.
+
+<h3><?= $this->escapeHtml($activity->getName()) ?></h3>
+
+<p><?= $this->escapeHtml($activity->getDescription()) ?></p>
+
+<br/>
+
+Het orgaan dat deze activiteit heeft aangemaakt kan worden bereikt via dit e-mailadres: <?= $this->escapeHtml($activity->getResourceOrgan()->getOrganInformation()['email']) ?>
```
Comment (jszanto) [inline module/Activity/view/email/activity_created_require_GEFLITST.phtml]: Not all activities have an organ associated with them, it might be better to use the email of the member whom created the activity.

File: module/Frontpage/src/Frontpage/Model/Poll.php [hunk 3]
```diff
@@ -148,7 +148,7 @@ public function getCreator()
     }
```
Comment (jszanto) [inline module/Frontpage/src/Frontpage/Model/Poll.php]: These files seem to be part of another pull request?

Comment (mstruijs) [inline module/Frontpage/src/Frontpage/Model/Poll.php]: Probably some auto-formatter.
